docs/getting-started.mdx

```diff
@@ -54,12 +54,12 @@ Easily share private network resources with public internet users, securely, wit
 
 The zrok public share is accessible from anywhere!
 
-zrok provides several different share types, including:
+zrok shares can be public or private, with different options for backend modes, including:
 
 * [public shares](./concepts/sharing-public.mdx) for [web services](./concepts/http.md) or [files](./concepts/files.md)
 * [private shares for web services or files](./concepts/sharing-private.mdx)
-* [tcpTunnel shares](./concepts/tunnels.md)
-* [udpTunnel shares](./concepts/tunnels.md)
+* [tcpTunnels](./concepts/tunnels.md)
```

Comment:
Suggest "TCP Tunnels" as link text because `tcpTunnels` resembles an explicit string that represents a valid backend mode.